Sync web platform tests for selection

rniwa opened the pull request:
#### 8a8928df8b492b671c16d7baba23855e55095a83
<pre>
Sync web platform tests for selection
<a href="https://bugs.webkit.org/show_bug.cgi?id=271039">https://bugs.webkit.org/show_bug.cgi?id=271039</a>

Reviewed by Tim Nguyen.

Updated web platform tests for selection as of 0d0f9ccf333f529e145c4c1dbfd8221ade20dc26.

* LayoutTests/imported/w3c/web-platform-tests/selection/anonymous/details-ancestor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/anonymous/details-ancestor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/anonymous/details-mutate-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/anonymous/details-mutate.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/anonymous/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/bidi/modify.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/bidi/modify.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/bidi/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/collapse-pre-linestart-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/collapse-pre-linestart-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/collapse-pre-linestart-2-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/collapse-pre-linestart-2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/collapse-pre-linestart-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/empty-elements-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/empty-elements.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/move-around-contenteditable-false-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/move-around-contenteditable-false.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/cefalse-on-boundaries-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/cefalse-on-boundaries.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/collapse-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/collapse.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/initial-selection-on-focus.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/initial-selection-on-focus.tentative_div-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/initial-selection-on-focus.tentative_span-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/modify.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/modify.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-non-primary-mouse-button.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-non-primary-mouse-button.tentative_middle-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-non-primary-mouse-button.tentative_secondary-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-primary-mouse-button.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-primary-mouse-button.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/selectall-and-find-svg-text-on-selectstart.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/selection-clip-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/selection-modify-around-input.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/selection-modify-around-textarea.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/selection-modify-line-boundary-around-empty-details.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/selection-modify-line-boundary-around-shadow.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/selection-modify-line-next-to-textarea.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/selection-modify-per-word-in-table-header-group.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/table.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/drag-disabled-textarea-shadow-dom-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/drag-disabled-textarea-shadow-dom.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/getRangeAt-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/selection/getRangeAt.html:
* LayoutTests/imported/w3c/web-platform-tests/selection/idlharness.window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/idlharness.window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify-extend-word-trailing-inline-block.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify-extend-word-trailing-inline-block.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify-line-flex-column.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify-line-flex-column.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify-line-flex-row.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify-line-flex-row.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify-line-grid-basic.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify-line-grid-basic.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/move-selection-range-into-different-root.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/move-selection-range-into-different-root.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/select-end-of-line-image.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/select-end-of-line-image.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-after-editinghost-removed-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-after-editinghost-removed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-after-textcontrol-removed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-after-textcontrol-removed_textControl=number-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-after-textcontrol-removed_textControl=password-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-after-textcontrol-removed_textControl=text-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-after-textcontrol-removed_textControl=textarea-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-in-shadow-after-the-shadow-removed.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-in-shadow-after-the-shadow-removed.tentative_mode=closed-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-in-shadow-after-the-shadow-removed.tentative_mode=open-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-shadow-dom-crash-print-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-shadow-dom-crash-print.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/test-iframe-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/selection/textcontrols/focus-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/textcontrols/focus.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/textcontrols/selectionchange-bubble-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/textcontrols/selectionchange-bubble.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/textcontrols/selectionchange-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/textcontrols/selectionchange.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/textcontrols/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/user-select-on-input-and-contenteditable-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/user-select-on-input-and-contenteditable.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/w3c-import.log: Added.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-non-primary-mouse-button.tentative_secondary-expected.txt: Added.
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-non-primary-mouse-button.tentative_secondary-expected.txt: Added.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-non-primary-mouse-button.tentative_middle-expected.txt: Added.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-non-primary-mouse-button.tentative_secondary-expected.txt: Added.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-primary-mouse-button.tentative-expected.txt: Added.
* LayoutTests/tests-options.json:

Canonical link: <a href="https://commits.webkit.org/276214@main">https://commits.webkit.org/276214@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/6bbc13f83d5a0bbd28b933470f25fbfc20a6da3f

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/43977 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/23045 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/46415 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/46626 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/40048 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/27027 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/20440 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/36273 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/44555 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/20136 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/37878 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/17291 "Passed tests") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/17627 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/38968 "Passed tests") | [✅ 🛠 wpe-skia](https://ews-build.webkit.org/#/builders/52/builds/2036 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/40215 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/39259 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/48204 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/18981 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/15554 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/43124 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/20366 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/38150 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/41842 "Passed tests") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/20575 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/6032 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/19993 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
Needs rebaselines, but this is fine otherwise

rniwa committed:
Sync web platform tests for selection
https://bugs.webkit.org/show_bug.cgi?id=271039

Reviewed by Tim Nguyen.

Updated web platform tests for selection as of 0d0f9ccf333f529e145c4c1dbfd8221ade20dc26.

* LayoutTests/imported/w3c/web-platform-tests/selection/anonymous/details-ancestor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/anonymous/details-ancestor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/anonymous/details-mutate-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/anonymous/details-mutate.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/anonymous/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/bidi/modify.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/bidi/modify.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/bidi/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/collapse-pre-linestart-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/collapse-pre-linestart-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/collapse-pre-linestart-2-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/collapse-pre-linestart-2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/collapse-pre-linestart-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/empty-elements-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/empty-elements.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/move-around-contenteditable-false-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/move-around-contenteditable-false.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/caret/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/cefalse-on-boundaries-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/cefalse-on-boundaries.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/collapse-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/collapse.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/initial-selection-on-focus.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/initial-selection-on-focus.tentative_div-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/initial-selection-on-focus.tentative_span-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/modify.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/modify.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-non-primary-mouse-button.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-non-primary-mouse-button.tentative_middle-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-non-primary-mouse-button.tentative_secondary-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-primary-mouse-button.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-primary-mouse-button.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/contenteditable/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/selectall-and-find-svg-text-on-selectstart.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/selection-clip-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/selection-modify-around-input.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/selection-modify-around-textarea.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/selection-modify-line-boundary-around-empty-details.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/selection-modify-line-boundary-around-shadow.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/selection-modify-line-next-to-textarea.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/selection-modify-per-word-in-table-header-group.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/table.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/crashtests/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/drag-disabled-textarea-shadow-dom-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/drag-disabled-textarea-shadow-dom.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/getRangeAt-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/selection/getRangeAt.html:
* LayoutTests/imported/w3c/web-platform-tests/selection/idlharness.window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/idlharness.window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify-extend-word-trailing-inline-block.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify-extend-word-trailing-inline-block.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify-line-flex-column.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify-line-flex-column.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify-line-flex-row.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify-line-flex-row.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify-line-grid-basic.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify-line-grid-basic.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/modify.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/move-selection-range-into-different-root.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/move-selection-range-into-different-root.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/select-end-of-line-image.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/select-end-of-line-image.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-after-editinghost-removed-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-after-editinghost-removed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-after-textcontrol-removed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-after-textcontrol-removed_textControl=number-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-after-textcontrol-removed_textControl=password-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-after-textcontrol-removed_textControl=text-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-after-textcontrol-removed_textControl=textarea-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-in-shadow-after-the-shadow-removed.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-in-shadow-after-the-shadow-removed.tentative_mode=closed-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-range-in-shadow-after-the-shadow-removed.tentative_mode=open-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-shadow-dom-crash-print-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/selection-shadow-dom-crash-print.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/test-iframe-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/selection/textcontrols/focus-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/textcontrols/focus.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/textcontrols/selectionchange-bubble-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/textcontrols/selectionchange-bubble.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/textcontrols/selectionchange-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/textcontrols/selectionchange.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/textcontrols/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/user-select-on-input-and-contenteditable-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/user-select-on-input-and-contenteditable.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/selection/w3c-import.log: Added.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-non-primary-mouse-button.tentative_secondary-expected.txt: Added.
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-non-primary-mouse-button.tentative_secondary-expected.txt: Added.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-non-primary-mouse-button.tentative_middle-expected.txt: Added.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-non-primary-mouse-button.tentative_secondary-expected.txt: Added.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-primary-mouse-button.tentative-expected.txt: Added.
* LayoutTests/tests-options.json:

Canonical link: https://commits.webkit.org/276214@main
